spec/models/pglogical_subscription_spec.rb

```diff
@@ -377,9 +377,7 @@ def with_an_invalid_remote_schema
       expect(remote_region.auth_key_configured?).to be true
 
       sub.delete
-      remote_region.reload
-
-      expect(remote_region.auth_key_configured?).to be false
+      expect(AuthToken.find_by_id(auth.id)).to be_nil
```

Comment:
@carbonin so, is removing the authentication record all that the `sub.delete` method does?  Or does it also change the result of `remote_region.auth_key_configured?` ?